systers_portal/templates/meetup/members.html

```diff
@@ -0,0 +1,63 @@
+{% extends "meetup/base.html" %}
+{% load staticfiles %}
+
+{% block meetup_location_page_content %}
+<div class="row mt40">
+  <h2 class="ml40">Organizers</h2>
+  <div class="col-sm-3">
+  {% for organizer in object.organizers.all %}
+    {% if organizer.profile_picture.name and organizer.profile_picture.name != "False" %}
+      <a href="{{ organizer.get_absolute_url }}">
+       <img src="{{ organizer.profile_picture.url }}" class="profile-pic img-square img-responsive crop_img" alt=""/>
+      </a>
+    {% else %}
+      <a href="{{ organizer.get_absolute_url }}">      
+        <img src="{%static 'img/default.png' %}" class="profile-pic img-square img-responsive crop_img"
+        alt="{{ organizer }} profile picture"/>
+      </a>
+    {% endif %}
+    <a href="{{ organizer.get_absolute_url }}">
+      {{ organizer.user.first_name }} {{ organizer.user.last_name }}
+    </a>
+    </div>
+    {% if forloop.counter|divisibleby:3 %}
+      </div>
+      <div class="row mt40">
+      <div class="col-sm-3">
+    {% else %}
+      <div class="col-sm-3">
+    {% endif %}
+  {% endfor %}
+  </div>
+</div>
+<div class="row">
+  <h2 class="ml40">Members</h2>
+  <div class="col-sm-3">
+  {% for member in object.members.all %}
+    {% if member.profile_picture.name and member.profile_picture.name != "False" %}
```

Comment:
Since it's the second time we repeat those lines of code that display the image and the name, it might be better to separate them into a snippet and then use them as `{% include ... with ...%}` (https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#include).